modules/client/src/client-common/utils.ts

```diff
@@ -117,3 +119,15 @@ export function votingModeFromContracts(votingMode: number): VotingMode {
       throw new InvalidVotingModeError();
   }
 }
+
+
+export function handleGraphQLError(error: Error, message: string) {
+  if (error instanceof ClientNotInitializedError) {
+    throw error;
+  } else if (error instanceof GraphQLClientError) {
+    const e = error as GraphQLClientError;
+    if (e.response.status < 500) {
+      throw new GraphQLError(message);
+    } 
+  }
```

Comment:
If the error is of another type, we do nothing?

Reply:
no, because it means we have a 5xx meaning there is a server error and we want to change the endpoint

Reply:
ok, ok... but then they should be on `failing` and the test should be selecting the `failing` ones. Don't you think?

Reply:
oh ok :+1:
Adding some comment to clarify this for future maintenability would be nide

Reply:
done, so resolving